src/components/Link.spec.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Link from './Link';
+
+const setup = (setupProps = {}) => {
+  const defaultProps = {
+    active: false,
+    onClick: jest.fn(),
+    children: 'Test link'
+  };
+  const props = { ...defaultProps, ...setupProps };
+  const wrapper = shallow(
+    <Link active={props.active} onClick={props.onClick}>
+      {props.children}
+    </Link>
+  );
+
+  return {
+    props,
+    wrapper
+  };
+};
+
+describe('Link', () => {
+  it('renders without crashing', () => {
+    const { wrapper } = setup();
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('renders a disabled button when active is true', () => {
+    const { wrapper } = setup({ active: true });
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('calls onClick() on click', () => {
```

Comment:
Maybe this one can be refactored to a better name such as:
```
describe('when button is clicked', () => {
  it('should trigger click event correctly', () => {
  })
})
```
So when you run the test, it'll print out a statement: 
```
when button is clicked
  should trigger click event correctly
```
It's more friendly for non-developer reading 😄